src/astra_frame_listener.cpp

```diff
@@ -73,7 +73,10 @@ void AstraFrameListener::onNewFrame(openni::VideoStream& stream)
 
     //ros::Time ros_now = ros::Time::now();
     rcl_time_point_value_t ros_now;
-    rcl_system_time_now(&ros_now);
+    if (rcl_system_time_now(&ros_now) != RCL_RET_OK)
+    {
+      ROS_ERROR("Failed to get current time");
```

Comment:
Is this code compiled against ROS1 core? Surprised to see ROS1 logging macros in here, I thought that ported packages were using printf (or frprintf to stderr) until the logging solution from @dirk-thomas is ready o_O